django/utils/timezone.py

```diff
@@ -52,7 +53,9 @@ def dst(self, dt):
 
 
 # UTC time zone as a tzinfo instance.
-utc = pytz.utc
+# We don't use datetime.timezone.utc here because its tzname is 'UTC+00:00'
```

Comment:
I'd replace this with: `# (Use utc = datetime.timezone.utc here when Python 3.5 isn't supported.)` (is that accurate?)